.travis.yml

```diff
@@ -0,0 +1,63 @@
+language: python
+python:
+- '3.6'
+- '3.7'
+dist: bionic
+services:
+- docker
+branches:
+  only:
+  - master
+  - develop
+
+addons:
+ apt:
+  update: true
+  sources:
+   - make 
+   - build-essential 
+   - libssl-dev 
+   - zlib1g-dev 
+   - libbz2-dev 
+   - libreadline-dev
+   - libsqlite3-dev 
+   - wget 
+   - curl 
+   - llvm 
+   - libncurses5-dev 
+   - libncursesw5-dev 
+   - xz-utils 
+   - tk-dev 
+   - libffi-dev
+   - liblzma-dev 
+   - automake 
+   - libtool 
+   - lsof 
+   - figlet 
+   - libsecp256k1-dev
+ snaps:
+  - name: go
+    confinement: classic
+
+before_install:
+- docker pull rabbitmq:3.7
+- sudo docker run -d -p 5672:5672 -h localhost --name rabbit rabbitmq:3.7
```

Comment:
how work this added?
```yml
- export PATH=/snap/bin/:$PATH
```

Reply:
I suffered finding how to use snap go instead of default go.
You finally made it!

Reply:
sadly It doesn't work. I don't know exactly what I did wrong, but I checked snap go installed and `$ go version` is 1.12.x.

But when I build icon rewardcalculator, its `Makefile` try to use default `go` whose version is 1.11.x.

T.T

Reply:
can you try this?
```yml
 - PATH=/snap/bin:$PATH make && make install
```